source/macos/AdaptiveCards/AdaptiveCards/Views/ACRTextField.swift

```diff
@@ -6,7 +6,7 @@ protocol ACRTextFieldDelegate: AnyObject {
 }
 
 class ACRTextField: NSTextField {
-    private enum Mode {
+    public enum Mode {
```

Comment:
If you are making it public, give a better descriptive name

Reply:
When you try to access it, it shows that it's a ACRTextField.Mode enum, so that should be descriptive enough, right?

Reply:
Oh ok!
I did not notice it was inside the class
This sounds good enough